src/main/java/com/visenze/datatracking/APIService.java

```diff
@@ -0,0 +1,22 @@
+package com.visenze.datatracking;
+
+import com.visenze.datatracking.data.DataTrackingResponse;
+import com.visenze.datatracking.data.EventsBody;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+import retrofit2.http.Query;
+
+public interface APIService {
+//    @Retry
+//    @Headers("Content-Type:application/json; charset=UTF-8")
+//    @GET("session")
+//    Call<DataTrackingResponse> getSession(@Query("timestamp") long timestamp, @Query("source") String source, @Query("uid") String uid, @Query("client_id") String cid);
+
+
```

Comment:
what about the single event image gif support ?

Reply:
ok added.